src/App.tsx

```diff
@@ -1,12 +1,16 @@
 import * as React from "react";
 import { Box, Text } from "@chakra-ui/layout";
+import Searchbar from "./components/InputField/input";
 
 const App: React.FC = () => {
   return (
-    <Box backgroundColor="red">
-      <Text color="orange" fontSize="100px">
-        Hello World
-      </Text>
+    <Box backgroundColor="white">
+      <Searchbar setSearch={function (value: React.SetStateAction<string | undefined>): void {
```

Comment:
What is the purpose of setSearch?

Reply:
Detects if the user is pressing the input to start a search.